scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.adoc

```diff
@@ -33,6 +33,32 @@ include::modules/ztp-creating-a-validator-inform-policy.adoc[leveloffset=+1]
 
 * xref:../../scalability_and_performance/ztp_far_edge/ztp-updating-gitops.adoc#ztp-updating-gitops[Upgrading GitOps ZTP]
 
+include::modules/ztp-using-pgt-to-configure-power-states.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref::../../../../scalability_and_performance/cnf-low-latency-tuning.html#node-tuning-operator-pod-power-saving-config_cnf-master[Optional: Power saving configurations]
```

Comment:
This resource should not be added here! It should be put as an Additional Resource in `modules/ztp-using-pgt-to-configure-power-saving-mode.adoc`